src/main/java/org/sagebionetworks/bridge/services/ExternalIdService.java

```diff
@@ -74,6 +77,7 @@ public final void setSubstudyService(SubstudyService substudyService) {
         return externalIdDao.getExternalIds(studyId, offsetKey, pageSize, idFilter, assignmentFilter);
     }
     
+    @SuppressFBWarnings("NP_NONNULL_PARAM_VIOLATION")
```

Comment:
It seems like this particular bug report is just, well, buggy. Instead of suppressing warnings piece-meal, it might be better to just use findbugs-exclude.xml to exclude this particular bug altogether.

```
  <Match>
    <!-- This bug pattern doesn't work correctly. -->
    <Bug pattern="NP_NONNULL_PARAM_VIOLATION" />
  </Match>
```

Reply:
I changed this as described, removed the annotation package.